Lib/multiprocessing/popen_fork.py

```diff
@@ -14,8 +14,10 @@ class Popen(object):
     method = 'fork'
 
     def __init__(self, process_obj):
-        sys.stdout.flush()
-        sys.stderr.flush()
+        if not sys.stdout.closed:
```

Comment:
Either change this to
`        if sys.stdout is not None and not sys.stdout.closed:`

or change the entire statement to
```
        try:
            sys.stdout.flush()
        except (AttributeError, ValueError):  # stdout None or closed.
            pass
```
and the equivalent for stderr.  I slightly prefer the latter (these exceptions should be rare), but if Antoine expresses a preference for the former, use that.